src/chrome/content/rules/Maybank-KE.co.th.xml

```diff
@@ -0,0 +1,35 @@
+<!--
+	For other Maybank coverage, see Maybank.com.xml
+
+
+	Non-functional subdomains:
+
+		- www.maybank-ke.co.th	(r)
```

Comment:
WFM, but has mixed content issues

Reply:
Those HTTP resources are available on HTTPS in my test, but doesn't load due to MCB. Anyhow, I updated the comment.